### ISSUE TITLE
Triton inference results lower than the corresponding .tlt model

### ISSUE THREAD

**Issue by gswmitrk** [triton-inference-server/server#5007]
I actually developed a gender model using TAO (I followed the train/prune/re-train steps) and ended with a .tlt model having an accuracy of 0.94 (I used evaluate on the val dataset as well as tao inference on a completely new datasets and the result was consistently 0.94 for both Male and Female classes). The problem is that the accuracy dropped to 0.8 for female and 0.91 for male when I tested the same model on Triton server (I exported the model using tao export). I used the VGG scaling at the inference time as the .tlt model used VGG19 as base model. Any explanation for that?

**Comment by GuanLuo** [comment]
Is VGG scaling also applied when you evaluate the accuracy outside Triton? Just want to make sure that the input data used is the same when comparing the accuracy. Another experiment is to compare accuracy of the exported model with / without Triton in case the exported model is not exactly equivalent to the model in original format.

If the accuracy drop still happens after eliminating the above variables, please provide reproduce steps to evaluate with / without Triton so we may investigate further.

**Comment by gswmitrk** [comment]
Hi @GuanLuo 
Actually I didn't use VGG scaling at the inference time with TAO (using the tao inference command). But when I call Triton without the VGG scaling the accuracy is even worse. I did the same testing with another model (developed following the same approach for age classification) and the result with Triton is lower for almost all the classes (6 age groups). I described the steps of my workflow as well as the results of the code in this post:
https://forums.developer.nvidia.com/t/converting-etlt-file-to-engine-for-jetson/230950/12?u=tarek.gas
I can also pass you the model file as well as the notebook to reproduce and further investigate the issue.
